Lib/test/test_pathlib.py

```diff
@@ -465,13 +465,18 @@ def test_parents_common(self):
         self.assertEqual(par[0], P('/a/b'))
         self.assertEqual(par[1], P('/a'))
         self.assertEqual(par[2], P('/'))
+        self.assertEqual(par[-1], P('/'))
```

Comment:
Maybe add a test for `par[-0]` (not sure if it should raise?)

A

Reply:
There's no distinction between integer `-0` and `0` in Python (but there is for floats):

```pythonrepl
>>> -0
0
>>> -0.0
-0.0
```

Reply:
I was remembering that slicing with minus 0 needed to be special cased, but this is precisely as `0 == -0`! 

Sorry for the noise, the fix looked good from a desk review.

A